packages/grafana-ui/src/themes/mixins.ts

```diff
@@ -2,36 +2,36 @@ import { CSSObject } from '@emotion/css';
 import { GrafanaTheme, GrafanaTheme2 } from '@grafana/data';
 import tinycolor from 'tinycolor2';
 
-export function cardChrome(theme: GrafanaTheme): string {
+export function cardChrome(theme: GrafanaTheme2): string {
   return `
-    background: ${theme.colors.bg2};
+    background: ${theme.colors.background.secondary};
     &:hover {
-      background: ${hoverColor(theme.colors.bg2, theme)};
+      background: ${hoverColor(theme.colors.background.secondary, theme)};
     }
-    box-shadow: ${theme.shadows.listItem};
-    border-radius: ${theme.border.radius.md};
+    box-shadow: ${theme.v1.shadows.listItem};
```

Comment:
I think this is basically `box-shadow: none` and could be written directly (it's probably not part of v2 theme)